Create addTransaction.handlebars

mekhtear opened the pull request.

mekhtear committed:
Create addTransaction.handlebars

MmyTohsaka commented on views/addTransaction.handlebars:
```diff
@@ -0,0 +1,51 @@
+<!-- views/addTransaction.hbs -->
```
Since this is a templated view that will be rendered inside of the `main.handlebars`, the document declaration stuff at the top won't be needed. We would only need whatever is going inside of the `<body>`. Removing everything except the `<main>` nest should make it valid.

MmyTohsaka commented on views/addTransaction.handlebars:
```diff
@@ -0,0 +1,51 @@
+
+                <!-- Method Input -->
+                <label for="methodInput">Payment Method</label><br>
+                <select name="methodInput" id="methodInput" required>
```
Methods will be rendered dynamically using the user's saved methods in their profile.

MmyTohsaka commented on views/addTransaction.handlebars:
```diff
@@ -0,0 +1,51 @@
+                <input type="number" name="amountInput" id="amountInput" step="0.01" placeholder="1000" required><br>
+
+                <!-- Category Input -->
+                <label for="categoryInput">Category</label><br>
```
If there is a way to offer a text input that also has drop down values, that would be ideal.

MmyTohsaka committed:
Update addTransaction.handlebars

I removed everything outside of the <main> tag since this will be a template loaded into the main handlebar.

MmyTohsaka reviewed (approved):
Fixed issue where HTML stuff outside of the `<main>` tag existed. Approving and merging now.
